hr_period/views/hr_period_view.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_hr_period_form" model="ir.ui.view">
+        <field name="name">hr.period.form</field>
+        <field name="model">hr.period</field>
+        <field name="arch" type="xml">
+            <form string="Payroll Periods">
+                <header>
+                    <button name="button_open" string="Open Period" type="object" states="draft"/>
+                    <button name="button_re_open" string="Re-Open Period" type="object" states="done"/>
+                    <button name="button_close" string="Close Period" type="object" states="open"/>
+                    <field name="state" widget="statusbar" statusbar_visible="draft,open,done"/>
+                </header>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name"/>
+                            <field name="fiscalyear_id" widget="selection"/>
+                            <label for="date_start" string="Duration"/>
+                            <div>
+                                <field name="date_start" nolabel="1" class="oe_inline"/> -
+                                <field name="date_end" nolabel="1" class="oe_inline"/>
+                            </div>
+                            <field name="date_payment"/>
+                        </group>
+                        <group>
+                            <field name="schedule_pay"/>
+                            <field name="type_id" domain="[('hr_period','=',True)]"/>
+                            <field name="number"/>
+                            <field name="company_id" widget="selection" groups="base.group_multi_company"/>
+                        </group>
+                    </group>
+                    <separator string="Payslips"/>
+                    <newline/>
+                    <field name="payslip_ids" nolabel="1"/>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="view_hr_period_tree" model="ir.ui.view">
+        <field name="name">hr.period.tree</field>
+        <field name="model">hr.period</field>
+        <field name="arch" type="xml">
+          <tree colors="blue:state == 'open';gray:state == 'done'" string="Payroll Period">
```

Comment:
OCA guideline: The `string` attribute in list views is deprecated (no longer displayed) since version 8.0 [[source](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#views)].

Same problem in:

- `views/hr_fiscalyear_view.xml`;
- `views/date_range_type_view.xml`.

---

OCA guideline: The `colors` attribute in list views is deprecated in favor of `decoration-{$name}` since version 9.0 [[source](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#views)].

Same problem in:
- `views/hr_fiscalyear_view.xml`.